sui_core/src/authority/authority_store.rs

```diff
@@ -986,6 +986,14 @@ impl<const ALL_OBJ_VER: bool, S: Eq + Serialize + for<'de> Deserialize<'de>>
         Ok(transaction)
     }
 
+    pub fn get_certified_transaction(
```

Comment:
previously this worked without modifying the authority store - did something change that requires this ?